app/policies/need_policy.rb

```diff
@@ -29,6 +29,14 @@ def destroy?
     create?
   end
 
+  def mark_unavailable?
```

Comment:
these could be aliases. 

eg. 
`alias mark_unavailable? show?`

Reply:
I suppose, it's not how the rest of them are done though

Reply:
true